addons/l10n_in_withholding/models/account_move.py

```diff
@@ -80,7 +80,23 @@ def _compute_l10n_in_warning(self):
                             domain=[('id', 'in', lines.ids)]
                         )
                 }
-                move.l10n_in_warning = warnings
+            if sections:
+                tds_tcs_applicable_lines = self.move_type == 'out_invoice' and self._get_tcs_applicable_lines(self.invoice_line_ids) or self.invoice_line_ids
+                warnings['tds_tcs_threshold_alert'] = {
+                        'message': sections._get_warning_message(),
+                        'action_text': _("View Journal Items(s)"),
+                        'action': tds_tcs_applicable_lines._get_records_action(
+                            name=_("Journal Items(s)"),
+                            target='current',
+                            views=[(self.env.ref('l10n_in_withholding.view_move_line_tree_tds_tcs_l10n_in_withholding').id, "list")],
```

Comment:
I guess then it's better to return dict then overriding the parameters. Using `_get_records_action` makes no sense here